src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const field = document.querySelector('.field');
+const buttons = document.querySelectorAll('.button');
+
+buttons.forEach(item => item.addEventListener('click', (e) => {
```

Comment:
use event delegation

Reply:
Omg, so I should have used the listener only for the container 😑, that was my first idea! 🧐